### PR TITLE
US120986 Quizzes: Add block alerts/disable pager and alerts checkbox

### PR DESCRIPTION
**Pull request by mostlyfabulous**
[US120986](https://rally1.rallydev.com/#/detail/userstory/438844250288?fdp=true): [ui] Add block alerts checkbox

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import '../d2l-activity-accordion-collapse.js';
+import './d2l-activity-quiz-disable-pager-and-alerts-editor.js';
 import './d2l-activity-quiz-disable-right-click-editor.js';
 import './d2l-activity-quiz-disable-right-click-summary.js';
+import '../d2l-activity-availability-dates-editor.js';
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js:5]: what's up with the addition of this line and the line below

**Review by wongvincent — COMMENTED** [review]
Could also add tests for the MobX `canEditDisablePagerAndAlerts` and `isDisablePagerAndAlertsEnabled` properties. That test file already has tests for `disableRightClick`, so it'll be pretty similar to that.

File: components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js [hunk 2]
```diff
@@ -11,5 +11,6 @@ export default {
 	"hintsAllowedSummary": "Hints allowed", // summary text when hints are enabled,
 	"passwordLabel": "Password", // Label for the quiz password text input field
 	"disableRightClickDescription": "Disable right click", // description for disable right click
-	"disableRightClickSummary": "Right clicks disabled" // summary text when right clicks are disabled
+	"disableRightClickSummary": "Right clicks disabled", // summary text when right clicks are disabled
+	"disablePagerAndAlertsDescription": "Block alerts and communication during quiz attempts",
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js:15]: nit. missing comment

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js (hunk 1 above)
Comment (mostlyfabulous) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js:5]: Ah must have been from rebasing my branch onto master. Removing those lines

File: components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js [hunk 3]
```diff
@@ -10,7 +10,8 @@ export default {
 	"hintsToolDescription": "Allow hints", // decription for hints tool
 	"hintsAllowedSummary": "Hints allowed", // summary text when hints are enabled,
 	"passwordLabel": "Password", // Label for the quiz password text input field
+  	"passwordSummary": "Password protected", // summary text when quiz has a password
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js:13]:
```suggestion
	"passwordSummary": "Password protected", // summary text when quiz has a password
```

**Review by wongvincent — DISMISSED** [review]

**Review by wongvincent — APPROVED** [review]

**Pull request merged**